### PR TITLE
OpenAPI Support

### PR DESCRIPTION
**Pull request by schnuerle**
## Explain pull request

Adding context and files to define CDS using OpenAPI.

## Is this a breaking change

* No, not breaking

## Impacted Spec

Which API(s) will this pull request impact?

* `Curbs`
* `Events`
* `Metrics`

## Additional context

This is useful in validating and viewing CDS data feeds. See issue #87 for more details.

### CODE REVIEW

**Comment by schnuerle** [comment]
@BigIdeas Make sure to click here and sign the OMF CLA: https://cla-assistant.io/openmobilityfoundation/curb-data-specification?pullRequest=100

File: curbs/curbs.yaml [hunk 1]
```diff
@@ -0,0 +1,1134 @@
+openapi: 3.1.0
+info:
+  title: Curbs
+  version: "1.0"
+  summary: "The Curbs API is a standard way for cities to digitally publish curb locations and regulations, which can be shared with the public and with companies using curb space. It defines curb policies, curb zones, spaces in zones, and areas around curbs, and is used by Events and Metrics."
```
Comment (schnuerle) [inline curbs/curbs.yaml:5]: Is a summary and a description valid. Got an error using the Swagger online viewer saying "info: should NOT have additional properties. additionalProperty: summary"

Comment (BigIdeas) [reply curbs/curbs.yaml:5]: I just checked on this and `summary` was introduced in v3.1.0 of the spec, and Swagger only supports up to v3.0.3.

It is probably worth removing `summary` and downgrading the `version` to `3.0.3` for Swagger compatibility. If you agree, there are some other differences between 3.1.0 and 3.0.3 that I can double check on (e.g., `exclusiveMinimum` being a number vs a boolean).

Comment (schnuerle) [reply curbs/curbs.yaml]: I think that's fine, so we can leave as is!

**Comment by BigIdeas** [comment]
> I do get these errors when in Stoplight. Do you see that too?

No, that's really strange, I wasn't seeing any errors in Stoplight but those were all real problems. I just pushed an update that should fix them.

**Comment by schnuerle** [comment]
This PR is going to get reworked to point to definitions in an external repo. 

https://github.com/openmobilityfoundation/cds-openapi/tree/v1.0.0-dev

**Comment by schnuerle** [comment]
@BigIdeas can you click this and sign the CLA please? https://cla-assistant.io/check/openmobilityfoundation/curb-data-specification?pullRequest=100

**Pull request merged**